lib/geokit/geocoders.rb

```diff
@@ -1,4 +1,5 @@
 require 'net/http'
+require 'openssl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.